recipe/meta.yaml

```diff
@@ -19,27 +22,27 @@ requirements:
   build:
     - python
     - pip
-    - mpich          # [unix]
```

Comment:
while here, we probably should pin mpich x.y as well.

Reply:
@minrk Do you mean the requirement should resolve to `mpich 3.2.*` everywhere?

Reply:
Yes, to ensure that an update doesn't cause a breakage.

Reply:
OK, done. Look at the way I define the `mpi` template var, and the commented-out `features` section. That way we would eventually specify MPI variants with environment variables `MPI_VARIANT="mpich 3.2.*"`, `MPI_VARIANT="openmpi 3.0.*"`.

Unless you raise further concerns, I'll wait for the Travis-CI builds to finish, then I'll merge.